ChromecastSampleApp/ChromecastSampleApp/Players/PlayerViewController.m

```diff
@@ -158,6 +158,7 @@ - (void)configureCastPlaybackViewBasedOnItem:(OOVideo *)item {
     NSString *videoDescription = item.itemDescription;
 
     dispatch_sync(dispatch_get_main_queue(), ^{
+      [[self.castPlaybackView subviews] makeObjectsPerformSelector:@selector(removeFromSuperview)];
```

Comment:
I think i see.  I was getting this functionality confused with when a customer can provide their own cast view.